projects/igniteui-angular/src/lib/tree/tree-node/tree-node.component.html

```diff
@@ -5,15 +5,17 @@
             <ng-container *ngTemplateOutlet="expandIndicatorTemplate, context: templateContext">
             </ng-container>
         </div>
-        <div class="igx-tree-node__select-marker" (click)="selected = !selected">
-            <ng-container *ngTemplateOutlet="selectMarkerTemplate, context: templateContext">
-            </ng-container>
-        </div>
+        <ng-container *ngIf="showSelectors">
```

Comment:
This can be moved on the `div`. No need for an extra wrapper element (even though ng-container is not generated).